docs/format/tile.rst

```diff
@@ -16,6 +16,15 @@ General notes:
 tilegrid.json
 -------------
 
+The file `tilegrid.json` contains lists of all tiles in the ROI and the configuration segments formed by those tiles. It also documents the membership relationship of tiles and segments.
```

Comment:
First mention of "ROI"

Reply:
True, and this sentence is no longer accurate.  Tilegrid lists all tiles, not just tiles in the ROI.

Reply:
Fixed description.